src/views/virtualmachines/details/tabs/network/copmonents/modal/NetworkInterfaceFormFields/NetworkInterfaceTypeSelect.tsx

```diff
@@ -0,0 +1,70 @@
+import * as React from 'react';
+
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import { FormGroup, FormSelect, FormSelectOption } from '@patternfly/react-core';
+
+type NetworkInterfaceTypeSelectProps = {
+  interfaceType: string;
+  setInterfaceType: React.Dispatch<React.SetStateAction<string>>;
+  networkName: string | undefined;
+};
+
+export const interfaceTypeTypes = {
```

Comment:
since I'm using here a state called `interfaceType` I wanted to name this with a name that will be easier to see that they are different, and moved this to constants.ts